services/email.js

```diff
@@ -0,0 +1,72 @@
+import {Microservices} from '../configs/microservices';
+import {SMTP} from '../configs/secrets';
+import rp from 'request-promise';
+import { isEmpty } from '../common.js';
+import SMTPConnection from 'smtp-connection';
+const log = require('../configs/log').log;
+
+export default {
+    name: 'email',
+    // At least one of the CRUD methods is Required
+    create: (req, resource, params, body, config, callback) => {
+        if (!params.from || !params.to || !params.subject || !params.message)
+            return callback('Error: Missing parameters', null);
+
+        let connection;
+        try {
+            connection = new SMTPConnection({
+                host: SMTP.host,
+                port: SMTP.port,
+                name: SMTP.clientName,
+                connectionTimeout: 4000,
+                opportunisticTLS: true,
+                tls: {
+                    rejectUnauthorized: false
+                }
+            });
+        }
+        catch (e) {
+            console.log(e);
+            return callback({message: 'Wrong SMTP configuration', error: e});
+        }
+
+        connection.on('error', (err) => {
+            console.log('ERROR on SMTP Client:', err);
+            if (process.env.NODE_ENV === 'test')
```

Comment:
Again, this should be replaced with `!(process.env.NODE_ENV) || process.env.NODE_ENV === 'development'`